hardware_service.py

```
@@ -194,29 +196,83 @@ def initialize_vfd_controller():
+                    120,
+                    json.dumps({
+                        "enabled": True,
+                        "port": port,
+                        "baudrate": baudrate,
+                        "channel": channel,
+                        "pan_id": pan_id,
+                        "status": "configured",
+                        "timestamp": datetime.now(timezone.utc).isoformat(),
+                    })
+                )
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)